src/libraries/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -531,21 +531,8 @@ private bool TrySkipHelper()
                 otherUtf8Text = stackalloc byte[JsonConstants.StackallocByteThreshold];
             }
 
-            ReadOnlySpan<byte> utf16Text = MemoryMarshal.AsBytes(text);
-            OperationStatus status = JsonWriterHelper.ToUtf8(utf16Text, otherUtf8Text, out int consumed, out int written);
-            Debug.Assert(status != OperationStatus.DestinationTooSmall);
-            bool result;
-            if (status > OperationStatus.DestinationTooSmall)   // Equivalent to: (status == NeedMoreData || status == InvalidData)
```

Comment:
Case in point: if I'm understanding this code correctly, this method will now throw when in the past it was returning `false`.